lib/sprinkler_web/live/dashboard_live.html.leex

```diff
@@ -0,0 +1,5 @@
+<h1>Temperature Control</h1>
+
+<%= for device <- @devices do %>
+  <%= live_component @socket, SprinklerWeb.TemperatureComponent, device: device %>
```

Comment:
I think you need an id

Reply:
why? 🤔

Reply:
Doesn't LiveView complain? I thought components needed an id to render